.github/workflows/tests.yaml

```diff
@@ -44,6 +43,8 @@ jobs:
         sudo sh -c "echo '127.0.0.1 testrequestbucket.localhost' >> /etc/hosts"
     - name: test and coverage
       run: yarn --silent coverage
+      env:
+        NODE_OPTIONS: "--tls-max-v1.2"
```

Comment:
as we need the options to pass the test:
* should it not be set instead in package.json instead, so the test can be run locally?
* do we need to do this kind of thing in the actual services (backbeat, cloudserver) as well, or is it really only affecting tests?

Reply:
For the moment it's only affecting tests , indeed to be run locally we can put this in the test command